src/pages/journal/journal.scss

```diff
@@ -10,6 +10,22 @@ page-journal {
   .refresh-button-icon {
     padding: 0 10px;
   }
+  #unauthenticated-mode-indicator {
+    text-align: center;
+    padding: 16px 0;
+    background: map-get($colors-gds, 'gds-red');
+    #unauth-text {
+      color: #FFFFFF;
+      font-family: SFProDisplay-Regular;
+      font-size: 20px;
+      font-weight: normal;
+      height: 25px;
```

Comment:
We shouldn't need these fixed height/width dimensions

Reply:
removed

Reply:
Can still see them?